server/master/src/main/java/org/apache/accumulo/master/state/SetGoalState.java

```diff
@@ -43,11 +40,11 @@ public static void main(String[] args) throws Exception {
     }
     SecurityUtil.serverLogin(SiteConfiguration.getInstance());
 
-    VolumeManager fs = VolumeManagerImpl.get();
-    Accumulo.waitForZookeeperAndHdfs(fs);
+    ServerInfo info = ServerInfo.getInstance();
```

Comment:
Need a follow on issue to look into passing more in and avoiding these calls to getInstance().

Reply:
Created #582